.eslintrc

```diff
@@ -1,21 +1,28 @@
 {
+  "root": true,
+  "reportUnusedDisableDirectives": true,
+  "parser": "@typescript-eslint/parser",
   "extends": [
     "plugin:react/recommended",
     "plugin:@typescript-eslint/recommended",
-    "prettier",
-    "prettier/@typescript-eslint"
+    // "prettier",
```

Comment:
I've disabled all prettier eslint rules for now, as it'd be too much work to fix it all at once right now, we can look into re-enabling them in the future.